src/JsonSchema/Enum.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace JsonSchema;
+
+class Enum extends \MabeEnum\Enum
+{
```

Comment:
You're right. My code makes sense, my comment doesn't. A user would extend `ConstraintError`, not `Mabe/Enum`, so my fear of the unfamiliar namespace being exposed was not correct.